app/controllers/api/functions.php

```diff
@@ -1417,7 +1461,6 @@
     ->label('sdk.auth', [APP_AUTH_TYPE_KEY])
     ->label('sdk.namespace', 'functions')
     ->label('sdk.method', 'createBuild')
-    ->label('sdk.description', '/docs/references/functions/create-build.md')
```

Comment:
Was this deleted on purpose?

Reply:
Yes, Eldad asked to hide it from the docs so that we can make breaking changes later.